error-503.html

```diff
@@ -7,125 +7,159 @@
+        </ul>
+        <ul class="main-nav__account-menu account-menu">
+          <li><a href="https://www.nhs.uk/Personalisation/Login.aspx">Log in</a></li>
+          <li><a href="https://www.nhs.uk/Personalisation/Registration.aspx">Create an account</a></li>
+        </ul>
       </div>
-    </header>
+    </nav>
+    <!-- endnoindex -->
+  </header>
+
+
+      <main id="content" role="main" class="util-nofocus">
+        <div class="breadcrumb">
+          <ol>
+            <li><a href="http://www.nhs.uk">Home</a></li>
+            <li><a href="https://www.nhs.uk/service-search">Services near you</a></li>
 
-    <main id="content" role="main">
-      <div class="main--inner">
+              <li class="breadcrumb__last">Find a pharmacy</li>
+
+          </ol>
+        </div>
+
+        <div class="page-section">
         <h1>Sorry, we are experiencing technical problems.</h1>
         <p class="error-message">Please try again later.</p>
       </div>
+
     </main>
 
-    <footer role="contentinfo">
-      <div class="global-footer">
-        <div class="global-footer--inner">
-          <h2 class="util-visuallyhidden">Support links</h2>
-          <ul class="link-list">
-            <li>
-              <a href="https://beta.nhs.uk/help/clinical-assurance">Clinical assurance</a>
-            </li>
-            <li>
-              <a href="https://beta.nhs.uk/help/cookies">Cookies</a>
-            </li>
-          </ul>
-        </div>
-      </div>
+    <footer role="contentinfo" class="footer">
+      <div class="footer__main-links">
+    <div class="footer__content">
+      <ul>
+        <li>
+          <a href="/">
+            <img class="footer__logo" src="https://assets.nhs.uk/images/nhs-logo.png" srcset="
+                https://assets.nhs.uk/images/nhs-logo.png 1x,
+                https://assets.nhs.uk/images/nhs-logo@2x.png 2x,
+                https://assets.nhs.uk/images/nhs-logo@3x.png 3x,"
+            />
+          </a>
+        </li>
+        <li><a href="http://www.nhs.uk/Pages/nhs-sites.aspx">NHS sites</a></li>
+        <li><a href="http://www.nhs.uk/Pages/about-us.aspx">About us</a></li>
+        <li><a href="http://www.nhs.uk/Pages/contact-us.aspx">Contact us</a></li>
+      </ul>
+      </ul>
+    </div>
+  </div>
+
+  <div class="footer__utility-links">
+    <div class="footer__content">
+      <ul>
+        <li><a href="http://www.nhs.uk/choices/pages/sitemap.aspx">Sitemap</a></li>
+        <li><a href="http://www.nhs.uk/aboutnhschoices/aboutnhschoices/accessibility/pages/accessibilitystatement.aspx">Accessibility</a></li>
+        <li><a href="http://www.nhs.uk/Pages/site-policies.aspx">Our policies</a></li>
+        <li>&copy; Crown Copyright</li>
+      </ul>
+    </div>
+  </div>
+
     </footer>
+    <script
+      src="https://code.jquery.com/jquery-1.12.4.min.js"
+      integrity="sha256-ZosEbRLbNQzLpnKIkEdrPv7lOy9C27hHQ+Xp8a4MxAQ="
+      cro
```

Comment:
I don't think the type attribute is required https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-type

Reply:
Removed.